src/app/services/chatbot.service.spec.ts

```
@@ -0,0 +1,126 @@
+    it('should return a directLine instance', async () => {
+      const result = await service.getDirectLine();
+      expect((window as any).WebChat.createDirectLine).toHaveBeenCalledWith({
+        domain: 'https://mock-directline.com/v3/directline',
+        token: 'mock-token',
+      });
+      expect(result).toBe('mockDirectLine');
+    });
+
+    it('should throw error if WebChat is not loaded', async () => {
+      const originalWebChat = (window as any).WebChat;
+      delete (window as any).WebChat;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)